rest_framework_jwt/views.py

```diff
@@ -23,6 +20,7 @@ class JSONWebTokenAPIView(APIView):
     authentication_classes = ()
     parser_classes = (parsers.FormParser, parsers.JSONParser,)
     renderer_classes = (renderers.JSONRenderer,)
+    response_payload_handler = staticmethod(jwt_response_payload_handler)
```

Comment:
using a static method there allows defining [custom response_payload_handler](https://github.com/erichonkanen/django-rest-framework-jwt/blob/jti/rest_framework_jwt/blacklist/views.py#L14) on a per-view basis w/o having to completely [rewrite](https://github.com/erichonkanen/django-rest-framework-jwt/blob/jti/rest_framework_jwt/views.py#L31) the post() method and not binding the method which would cause the view object to be passed into the method. Since the data being returned by that doesn't depend on the view object and lives in the utils file, using static method there seemed like good use case (ran that by #python as well)..

tldr - it didn't work w/o redefining entire post method if I didn't use static method, saves some repeat